README.md

```diff
@@ -1 +1,146 @@
-# Awsome-book-ES6
\ No newline at end of file
+# Awsome-book-ES6
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://media.wbur.org/wp/2022/12/cover-e1670599123274-1000x661.jpg" alt="logo" width="160"  height="auto" />
+  <br/>
+   <br/>
+
+  <h1><b> Awesome-Books 📚📚📚</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<br>
+
+<!-- DEPLOYMENT -->
+
+### This project goes live [here](https://yemisirach.github.io/Awsome-book-ES6/) 🚀 .
+
+<br>
+
+<!-- DEPLOYMENT -->
+<!-- ### Have a look at the project's video presentation [here](https://www.loom.com/share/1f7de2b9345a4d3988c6327952899d30) 📽️ . -->
+
+<br>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books ES6] <a name="about-project"></a>
+
+**Awesome Books** is a pair programming project for saving book details like an online library.
+
+## 🛠 Built With <a name="built-with"></a>
+
+-Client
+
+- **This Project has a Readme.md file**
+- **This Project has a gitignore file**
+- **This projects follows all Microverse Linters**
+- **This project takes user inputs of Book title and Authors**
+- **This project displays the input for Book details dynamically on the DOM**
+- **This project saves the Book details insode the Browser**
```

Comment:
- [ ] I would suggest that rather than mentioning what the project "has" you list the technologies you used, e.g. HTML, CSS, JavaScript. And you can define a header "Key features" where you can list what people can "do" with your project, e.g. Create a book, See a list of books and Remove books. 🤓 🔧